xmake.lua

```diff
@@ -12,6 +12,7 @@ target("traffico")
     set_default(true)
     includes("api")
     add_deps("api")
+    add_deps("bpf")
```

Comment:
Leaving a note for the future myself.

This `add_deps` should not be needed because the "api" target already brings it as a transitive dependency.